<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminlogin</class>
 <widget class="QDialog" name="adminlogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>526</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/image/.qtcreator/thumb-1920-1398872.jpg);</string>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: light gray;
color: rgb(231, 255, 247);
text-decoration: underline;
font: 25pt &quot;Segoe Print&quot;</string>
   </property>
   <property name="text">
    <string>CinemaXx</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removebtn1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>remove</string>
   </property>
  </widget>
  <widget class="QLabel" name="photolabel1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>161</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/KIIT/Downloads/avatar.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="photolabel2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>171</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/KIIT/Downloads/nemo.jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="photolabel3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>171</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/KIIT/Downloads/angry bird.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="photolabel4">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>60</y>
     <width>171</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/KIIT/Downloads/1.jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QScrollBar" name="horizontalScrollBar">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>750</y>
     <width>661</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>835</x>
     <y>137</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>ADD NEW MOVIE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BACK">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removebtn3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>320</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removebtn2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>320</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removebtn4">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>320</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>remove</string>
   </property>
  </widget>
  <widget class="QLabel" name="language1">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>363</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Language:</string>
   </property>
  </widget>
  <widget class="QLabel" name="name1">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>321</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Name of Movie:</string>
   </property>
  </widget>
  <widget class="QLabel" name="duration1">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>342</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Duration:</string>
   </property>
  </widget>
  <widget class="QLabel" name="duration2">
   <property name="geometry">
    <rect>
     <x>191</x>
     <y>342</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Duration:</string>
   </property>
  </widget>
  <widget class="QLabel" name="name2">
   <property name="geometry">
    <rect>
     <x>191</x>
     <y>321</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Name of Movie:</string>
   </property>
  </widget>
  <widget class="QLabel" name="language2">
   <property name="geometry">
    <rect>
     <x>191</x>
     <y>363</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Language:</string>
   </property>
  </widget>
  <widget class="QLabel" name="name3">
   <property name="geometry">
    <rect>
     <x>391</x>
     <y>321</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Name of Movie:</string>
   </property>
  </widget>
  <widget class="QLabel" name="duration3">
   <property name="geometry">
    <rect>
     <x>391</x>
     <y>342</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Duration:</string>
   </property>
  </widget>
  <widget class="QLabel" name="language3">
   <property name="geometry">
    <rect>
     <x>391</x>
     <y>363</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Language:</string>
   </property>
  </widget>
  <widget class="QLabel" name="duration4">
   <property name="geometry">
    <rect>
     <x>591</x>
     <y>342</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Duration:</string>
   </property>
  </widget>
  <widget class="QLabel" name="name4">
   <property name="geometry">
    <rect>
     <x>591</x>
     <y>321</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Name of Movie:</string>
   </property>
  </widget>
  <widget class="QLabel" name="language4">
   <property name="geometry">
    <rect>
     <x>591</x>
     <y>363</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Language:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
